scripts/zones/Caedarva_Mire/npcs/qm8.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Area: Caedarva Mire
+-- NPC:  qm8
+-- Gives Lamian Fang Key
+-----------------------------------
+package.loaded["scripts/zones/Caedarva_Mire/TextIDs"] = nil
+-----------------------------------
+require("scripts/zones/Caedarva_Mire/TextIDs")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local fangKeyTimer = player:getVar("[TIMER]Lamian Fang Key")
+
+    if fangKeyTimer < os.time() then
+        if player:getFreeSlotsCount() > 0 and player:hasItem(2219) == false then
+            player:addItem(2219,1)
+            player:messageSpecial(ITEM_OBTAINED,2219)
+            player:setVar("[TIMER]Lamian Fang Key",getConquestTally())
```

Comment:
Not sure why `getConquestTally()` is used, this would result in one free key a conquest tally, rather than every vanadiel day, unless you've confirmed this to not be true.

`A character can obtain a free Lamian Fang Key once per Vana'diel day (resets at 0:00 game time) from a ??? in Caedarva Mire (H/I-7), in a small crevice behind a Jnun.`
http://ffxiclopedia.wikia.com/wiki/Lamian_Fang_Key

Reply:
oh crap... i dorked that... pulled old code from wrong place...

Reply:
Yeah, thanks for catching that... I spent the better part of the day trying and trying to get npc_utils to work before finally giving up and grabbing my old code and just didn't pay enough attention to it. it's all good now.